src/main/java/nl/praegus/fitnesse/responders/ToolTip/ToolTips.java

```diff
@@ -0,0 +1,74 @@
+package nl.praegus.fitnesse.responders.ToolTip;
+
+import java.io.*;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.*;
+import java.util.jar.JarEntry;
+import java.util.jar.JarFile;
+import java.util.stream.Collectors;
+
+public class ToolTips {
+
+    private List<String> toolTips = new ArrayList<>();
+    public String tooltipTargetDirectory = "/TooltipData";
+    public String bootstrapTargetDirectory =  "/TooltipData/fitnesse/resources/bootstrap-plus/txt/toolTipData.txt";
+
+    private void addToolTips(){
+        File[] dirs = new File(System.getProperty("user.dir")+ tooltipTargetDirectory).listFiles();
+        for (File dir : dirs){
+            // check if File in list is directory so we wont try to listfiles from a file
+            if(dir.isDirectory()){
+                for (File tooltipFile : new File(dir.getPath()).listFiles()) {
+                    //check if the file is actual tooltips.txt file so we arent reading any other txt files
+                    if (tooltipFile.getName().toLowerCase().equals("tooltips.txt")) {
```

Comment:
Waarom ga je hier alle files aflopen en kijken of er toevallig eentje tooltips.txt heet? kun je niet beter gewoon tooltips.txt openen en de exceptie afvangen als die er niet is? Lijkt me een stuk efficienter